<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Default Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>530</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>321</width>
     <height>460</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutLeft">
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="metadata_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Metadata</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayoutMetadata">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="base_uuid_label">
        <property name="text">
         <string>Base URL:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="base_uuid_edit">
        <property name="toolTip">
         <string>Website of the data provider</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="title_label">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="title_edit">
        <property name="toolTip">
         <string>Human friendly title of the image</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="platform_label">
        <property name="text">
         <string>Platform:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="platform_combo_box">
        <item>
         <property name="text">
          <string>satellite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>aircraft</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UAV</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>balloon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>kite</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="sensor_label">
        <property name="text">
         <string>Sensor:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="sensor_edit">
        <property name="toolTip">
         <string>Image aquisition device (eg. camera, radar, etc)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="sense_start_label">
        <property name="text">
         <string>Sense start</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateTimeEdit" name="sense_start_edit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="sense_end_label">
        <property name="text">
         <string>Sense end</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDateTimeEdit" name="sense_end_edit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="provider_label">
        <property name="text">
         <string>Provider:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="provider_edit">
        <property name="toolTip">
         <string>Provider/owner of the OIN bucket</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="contact_label">
        <property name="text">
         <string>Contact:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="contact_edit">
        <property name="toolTip">
         <string>Name and email address of the data provider</string>
        </property>
        <property name="cursorPosition">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="options_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Options</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutOptions">
      <item>
       <widget class="QCheckBox" name="license_check_box">
        <property name="text">
         <string>Accept OAM free license terms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="reproject_check_box">
        <property name="text">
         <string>Reproject to EPSG:3857</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="oam_catalog_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OAM Catalog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="hot_oam_catalog_check_box">
      <property name="text">
       <string>Use HOT OAM Catalog</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayoutCatalog">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="catalog_url_label">
        <property name="text">
         <string>Catalog URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="catalog_url_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>40</y>
     <width>331</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutRight">
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="image_storage_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Image Storage</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayoutStorage">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="aws_s3_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>AWS S3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="aws_bucket_name_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Bucket name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="aws_bucket_name_edit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Bucket name which is not listed above </string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="aws_key_id_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Key-id</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="aws_key_id_edit">
        <property name="toolTip">
         <string>AWS access key id</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="aws_secret_key_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Secret key</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="aws_secret_key_edit">
        <property name="toolTip">
         <string>AWS secret access key</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="google_drive_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Google Drive</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="google_client_secret_file_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Client Secret File</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="google_client_secret_file_edit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Bucket name which is not listed above </string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="google_application_name_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Application Name</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="google_application_name_edit">
        <property name="toolTip">
         <string>AWS access key id</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="dropbox_label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Drop Box</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="dropbox_access_token_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Access Token</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="dropbox_access_token_edit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="default_storage_type_label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Default StorageType</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="default_storage_combo_box">
        <item>
         <property name="text">
          <string>AWS S3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Google Drive</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dropbox</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="metadata_storage_label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Metadata Storage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="metadata_storage_check_box">
      <property name="text">
       <string>Same as Image Storage</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="meta_bucket_name_label">
        <property name="text">
         <string>Bucket Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="meta_bucket_name_edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="meta_key_id_label">
        <property name="text">
         <string>Key-id</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="meta_key_id_edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="meta_secret_key_label">
        <property name="text">
         <string>Secret Key</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="meta_secret_key_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>691</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>691</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
